zuul.d/jobs.yaml

```diff
@@ -22,10 +22,26 @@
     nodeset: centos-8-1vcpu
 
 - job:
-    name: ansible-tox-molecule
-    parent: tox-molecule
+    name: ansible-tox-docs
+    parent: tox-docs
+    nodeset: centos-8-1vcpu
+
+- job: &_stub_molecule
+    name: _stub_molecule
+    # Stub job used to define only properties specific to molecule jobs,
+    # expected to decorate other job definitions via anchors. To avoid overrides
+    # it does not have: nodeset, abstract, description
     pre-run: playbooks/ansible-tox-molecule/pre.yaml
     post-run: playbooks/ansible-tox-molecule/post.yaml
+    timeout: 5400  # 1.5h as they are expected to be slower by nature
+
+- job:
+    <<: *_stub_molecule
+    name: ansible-tox-molecule
+    description: |
+      Generic molecule jobs that does not specify which python version will
+      be using, recommended when that is not important.
```

Comment:
You maybe should set job.abstract = true here, so people don't use it directly

Reply:
There is nothing wrong in using this job, in fact is recommented for most users. The most specific ones are needed for special cases, like testing molecule itself. But when someone wants to just run molecule, they usually do not care which python version is used. I would have added abstract if I wanted to avoid its use.